docker-compose.yaml

```diff
@@ -62,7 +62,7 @@ services:
 
   admin:
     image: node:11-alpine
-    command: yarn run prod
+    command: yarn run start
```

Comment:
just `yarn start` or... if yarn is making problems, `npm start` should be ok.

Reply:
Fixed